generators/client/templates/_package.json

```diff
@@ -8,6 +8,7 @@
     "<%= MAIN_SRC_DIR %>bower_components"
   ],
   "devDependencies": {
+    "bower": "1.7.9",
```

Comment:
I dont like to have this in the dev dependency as it will install a local bower. our recommendation is to use global npm and global bower. A jhipster projects node_modules folder is already too hug I wouldn't want to add un necessary dependencies to it